### PR TITLE
Added validation to Employee and Meeting Service classes

### PR DESCRIPTION
**Pull request by JuliiaRak**

### CODE REVIEW

File: src/main/java/com/solvd/domain/Employee.java [hunk 1]
```diff
@@ -17,4 +16,14 @@ public class Employee {
...
+
+    public Employee(String firstName, String lastName, String email, String phoneNumber, String position, Date hireDate, long salary) {
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.email = email;
+        this.phoneNumber = phoneNumber;
+        this.position = position;
+        this.hireDate = hireDate;
+        this.salary = salary;
+    }
```
Comment (xd6d) [inline src/main/java/com/solvd/domain/Employee.java:28]: No usages of this constructor

File: src/main/java/com/solvd/domain/Meeting.java [hunk 2]
```diff
@@ -19,4 +17,13 @@ public class Meeting {
...
     private Employee employee;
+
+    public Meeting(Date meetingDateTime, Date inquiryDate, String meetingStatus, RealEstate realEstate, Client buyer, Employee employee) {
+        this.meetingDateTime = meetingDateTime;
+        this.inquiryDate = inquiryDate;
+        this.meetingStatus = meetingStatus;
+        this.realEstate = realEstate;
+        this.buyer = buyer;
+        this.employee = employee;
+    }
```
Comment (xd6d) [inline src/main/java/com/solvd/domain/Meeting.java:28]: Only 1 usage in Main, so mb we can do without it

File: src/main/java/com/solvd/service/MeetingService.java [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
 
 import java.util.List;
 import java.util.Optional;
 
 public interface MeetingService {
     void create(Meeting meeting, Long realEstateId, Long buyerId, Long employeeId);
     void deleteById(long id);
-    void update(Meeting meeting, Long realEstateId, Long buyerId, Long employeeId);
+    void update(Meeting meeting, Long realEstateId, Long buyerId, Long employeeId) throws EntityNotFoundException;
     Optional<Meeting> getById(long id);
```
Comment (xd6d) [inline src/main/java/com/solvd/service/MeetingService.java]: Remove Optional

File: src/main/java/com/solvd/service/MeetingService.java
Comment (DenysKulikov) [inline src/main/java/com/solvd/service/MeetingService.java:1]: Remove Optional

File: src/main/java/com/solvd/service/impl/MeetingServiceImpl.java
Comment (DenysKulikov) [inline src/main/java/com/solvd/service/impl/MeetingServiceImpl.java:1]: Remove Optional

**Review by DenysKulikov — CHANGES_REQUESTED** [review]
Remove Optional in services

**Review by DenysKulikov — COMMENTED** [review]
Sorry, testing comments

**Review by DenysKulikov — APPROVED** [review]

**Pull request merged**